0000-kubect-via-scipian.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: kubectl_via_scipian
+- Start Date: 2019-07-01
+- RFC PR: [scipian/rfcs#0000](https://github.com/scipian/rfcs/pull/0000)
+- Community Issue: [scipian/community#0000](https://github.com/scipian/community/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Enable users to interact with their kubernetes clusters via Scipian. Commands desired are:
+
+```
+- kubectl create -f .
+- kubectl apply -f .
+- kubectl get <resource>
+- kubectl describe <resource> 
+- kubectl logs <resource>
+- kubectl scale --replicas=<num> <resource>
+```
+
```

Comment:
Let's break these down into just the commands, we should design this around the commands and resources you want access to in the desired cluster. For each command above, provide the functionality you hope to gain with access to the corresponding command. Also, we should think about auto-scaling and how that affects some of the resource access. The other thing to think about is how much of this is just observability into the desired cluster. Which could be a separate RFC by breaking observability into it's pillars:

1. Structured Logs
2. Metrics
3. Tracing
4. Errors